spec/factories/miq_request.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```

Comment:
Factories should live in the core repo to avoid conflicts. But you probably won't need it anyway, there's some :sparkles: magic :sparkles: around them that creates them on-the-fly if they're missing. Just pass these arguments to the `.create`.